src/components/Dashboard/Dashboard.tsx

```diff
@@ -108,19 +95,33 @@ export const Dashboard = () => {
         )}
         {formVisible && (
           <form className={style.form} onSubmit={handleSubmit}>
-            <Row fullWidth justifyStart>
-              <label htmlFor="activity">
-                aktivita:
-                <input
-                  id="activity"
+            <Row fullWidth justifyStart flexCol>
+              <TextField
+                type="text"
+                label="aktivita"
+                placeholder="běhání"
+                className={style.input}
+                onChange={(e) => setNewActivity(e.target.value)}
+                autoFocus
+              />
+              <div className={style.inputWrapper}>
+                <TextField
+                  type="number"
+                  label="množství"
+                  placeholder="30"
+                  className={style.input}
+                  onChange={(e) => setNewAmount(+e.target.value)}
```

Comment:
This is my personal opinion, but let's follow it :D : 

The only good implicit conversion is to boolean with `!!`. If you want to convert to a number, use `Number(x)`. If you want to convert to a string, avoid using \`${stringToConvert}\` and rather go for either `.toString()` or `String(x)`. The latter will not crash if `x` is undefined, so I recently started preferring it :)).